sormas-api/src/main/resources/enum.properties

```diff
@@ -2281,3 +2281,14 @@ Pathogen.REOVIRUS = Reovirus
 Pathogen.ROTAVIRUS = Rotavirus
 Pathogen.TT_HEPATITIS = TT hepatitis
 Pathogen.OTHER = Other
+# SelfReportType
+SelfReportType.CASE=Case
+SelfReportType.CONTACT=Contact
+# SelfReportInvestigationStatus
+SelfReportInvestigationStatus.IN_PROGRESS=In progress
+SelfReportInvestigationStatus.PENDING=Pending
+SelfReportInvestigationStatus.COMPLETED=Completed
+SelfReportInvestigationStatus.REJECTED=Rejected
+# SelfReportProcessingStatus
+SelfReportProcessingStatus.UNPROCESSED=Unprocessed
+SelfReportProcessingStatus.PROCESSED=Processed
```

Comment:
Only the translation related to the Self Report user rights and SELF_REPORT title are still missing. Besides this everything is fine and tested.

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/48b25a5c-99d3-4002-a4f7-ac83bcc6d905)